hubapi/README.MD

```diff
@@ -0,0 +1,75 @@
+`HubAPI`
+The HubAPI supports operations for listing jina hub images
+and pushing images to the hub. These are authorised using the
+hub authorizer. We use MongoDB for storing and retrieving
+information related to hub. This document elaborates on the functioning
+of each of these APIs
+1. `Hub List` API
+List API provides the user the ability to query jina images. Under the hood, the implementation uses a Mongodb Handler to connect to the database & insert documents in the collection.
```

Comment:
*List API provides the user the ability to query jina images*

Simplify to:

*The list API lets users query **Jina** images*